showcase/src/app/shared/example-viewer/example-viewer.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit, Input } from "@angular/core";
+import { HttpErrorResponse } from "@angular/common/http";
+import { FileService } from "./file.service";
+
+@Component({
+	selector: "example-viewer",
+	templateUrl: "./example-viewer.html",
+	styleUrls: ["./example-viewer.scss"],
+	providers: [FileService]
+})
+export class ExampleViewerComponent implements OnInit {
+	@Input() public extensions: Array<string> = ["HTML", "TS", "CSS"];
+	@Input() public filesPath: string;
+	@Input() public title: string;
+
+	public filesContent: Array<Object> = [];
+	public showSource: boolean = false;
+
+	public constructor(private service: FileService) {}
+
+	public ngOnInit(): void {
+		this.fetchFiles();
+	}
+
+	public fetchFiles(): void {
+		this.extensions.forEach((extension: string) => {
+			this.service
+				.fetchFile("/assets/examples/" + this.filesPath + "." + extension.toLowerCase())
+				.subscribe(
+					(data: string) => this.addFileContent({ extension: extension, file: data }),
+					(error: HttpErrorResponse) => console.log(error)
```

Comment:
One small change: just replace this console.log by the StarkLogging service?

Normally we should always use the StarkLogging service. We should use `console` In case it is not possible to use the Stark service or it is really necessary.

Reply:
Finally we'll leave the `console` in there because there is an issue with the Stark mocks. Tracked in #484